README.md

```diff
@@ -4,19 +4,19 @@ A mocking library based on the Compiler APIs (Roslyn + Mocks)
 
 ## Overview
 
-There are great mocking libraries out there, like [Moq](https://github.com/moq/moq) and [NSubstitute](http://nsubstitute.github.io/), so why did I decide to create YAML (yet another mocking library) in 2015? There are essentially two reasons.
+There are great mocking libraries out there, like [Moq][1] and [NSubstitute][2], so why did I decide to create YAML (yet another mocking library) in 2015? There are essentially two reasons.
```

Comment:
I'd like to keep the links at the place where they're referenced and not footnotes, so please change those back.

Reply:
Acknowledged, I'll be reverting the links to their previous position!